zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -569,10 +573,9 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             red_chunks = reduction_chunker(
                 idx, mode="exact", stride_start_offset_in_unit=stride_start_offset_in_unit
             )
+            reduction_dst = None if dst is None else dst.with_procs(read_procs=(), write_procs=())
             tasks_reduce = [
-                Copy().make_task(
-                    src=dst_temp, dst=dst.with_procs(read_procs=(), write_procs=()), idx=red_chunk
-                )
+                Copy().make_task(src=dst_temp, dst=reduction_dst, idx=red_chunk)
```

Comment:
This is weird - is any of this change necessary? We've already asserted that `dst is not None` above.

Reply:
Yeah this was left over from earlier work, and I didn't catch it.  You are right.